dashboards/etcd/etcd.json

```diff
@@ -122,7 +122,7 @@
               "nrqlQueries": [
                 {
                   "accountId": 0,
-                  "query": "SELECT (average(etcd_debugging_mvcc_db_total_size_in_bytes) * cardinality(etcd_debugging_mvcc_db_total_size_in_bytes)) / 1000 / 1000  as 'DB Total Size (MB)' FROM Metric where instrumentation.name = 'remote-write' facet instance SINCE 60 MINUTES AGO UNTIL NOW"
+                  "query": "SELECT (average(etcd_mvcc_db_total_size_in_bytes) * cardinality(etcd_mvcc_db_total_size_in_bytes)) / 1000 / 1000  as 'DB Total Size (MB)' FROM Metric where instrumentation.name = 'remote-write' facet instance SINCE 60 MINUTES AGO UNTIL NOW"
```

Comment:
i would update this to make it backwards compatible with older versions:

```
SELECT 
(average(etcd_mvcc_db_total_size_in_bytes) * cardinality(etcd_mvcc_db_total_size_in_bytes)) 
OR (average(etcd_debugging_mvcc_db_total_size_in_bytes) * cardinality(etcd_debugging_mvcc_db_total_size_in_bytes)) / 1000...
```

please check the syntax on this, but the logic is there

Reply:
Thank you @thezackm , updated query as suggested.

<img width="1264" alt="image" src="https://user-images.githubusercontent.com/71118025/199101712-4f7b9b83-ecf5-4468-b388-973bb8f8f5f9.png">